apps/client/src/app/components/rule/rule-settings-dialog/rule-settings-dialog.html

```diff
@@ -1,76 +1,126 @@
 <div mat-dialog-title>{{ data.rule.name }}</div>
 
 <div class="py-3" mat-dialog-content>
-  <div
-    class="w-100"
-    [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMin }"
-  >
-    <h6 class="mb-0">
-      <ng-container i18n>Threshold Min</ng-container>:
+  @if (
+    data.rule.configuration.thresholdMin && data.rule.configuration.thresholdMax
+  ) {
+    <div
+      class="w-100"
+      [ngClass]="{
+        'd-none':
+          !data.rule.configuration.thresholdMin ||
+          !data.rule.configuration.thresholdMax
+      }"
+    >
+      <h6 class="mb-0">
+        <ng-container i18n>Threshold</ng-container>:
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMin | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMin }}
+        }
+        -
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMax | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMax }}
+        }
+      </h6>
+      @if (data.rule.configuration.threshold.unit === '%') {
```

Comment:
Can you wrap the slider including labels with `<div class="align-items-center d-flex w-100">`